scripts/development/m3_stack/m3dbnode.yml

```diff
@@ -56,6 +56,8 @@ db:
   cache:
     series:
       policy: lru
+    postingsList:
```

Comment:
how would you feel about moving this to the `index:` section?

Reply:
Don't have super strong feelings about it, but from an operator perspective seems nice to be able to tune all the caches in one place.

Reply:
actually now I'm torn :/ maybe I'll just do index since thats where your gut said it should go. Should be ok once this P.R lands #1371 configs should be a lot simpler

Reply:
although we don't currently have anything under the index field...

Reply:
Nah we have the maxconcurrentquery knob there already.

It makes more sense to me under `index:` cause the rest of the code, e.g. IndexOptions are grouped as such too

Reply:
Ok sure will move it there

Reply:
@prateek Although to be fair the YAML config is for humans, so I don't think how the rest of the code is organized should be the decision maker. Its more about where it makes sense for an operator to look at and modify it

Reply:
I think I might leave as is. We're gonna keep adding caches I think its nice all in one place.

Reply:
Sure, I don't feel strongly either way